src/resources/api/auth.js

```diff
@@ -0,0 +1,23 @@
+import api from "./config";
+
+let apiPrefix = "/badaso-api";
+
+export default {
+  register(data) {
+    let ep = apiPrefix + "/module/lms/v1/auth/register";
+    let response = api.post(ep, data);
+    response.then((res) => {
+      if (res.data.accessToken) {
+        let token = res.data.accessToken;
+        localStorage.setItem("token", token);
+        let date = new Date();
+        let timeNow = date.getTime();
+        localStorage.setItem(
+          window.btoa("tokenAccessTime"),
+          window.btoa(timeNow)
+        );
+      }
+    });
+    return response;
+  },
+};
```

Comment:
nit: need some whitespaces to be able to "breathe"
nit label means you're free to not implement it, just resolve the issue if you prefer to do so